wagtailio/templates/includes/svg_sprites.html

```diff
@@ -15,4 +16,14 @@
.1804 18.5 27.1794 18.5 27.1785C18.5 27.1776 18.5 27.1767 18.5 27.1758C18.5 27.1749 18.5 27.174 18.5 27.173C18.5 27.1721 18.5 27.1712 18.5 27.1703C18.5 27.1694 18.5 27.1685 18.5 27.1675C18.5 27.1666 18.5 27.1657 18.5 27.1648C18.5 27.1639 18.5 27.163 18.5 27.162C18.5 27.1611 18.5 27.1602 18.5 27.1593C18.5 27.1584 18.5 27.1575 18.5 27.1566C18.5 27.1556 18.5 27.1547 18.5 27.1538C18.5 27.1529 18.5 27.152 18.5 27.1511C18.5 27.1501 18.5 27.1492 18.5 27.1483C18.5 27.1474 18.5 27.1465 18.5 27.1456C18.5 27.1447 18.5 27.1437 18.5 27.1428C18.5 27.1419 18.5 27.141 18.5 27.1401C18.5 27.1392 18.5 27.1382 18.5 27.1373C18.5 27.1364 18.5 27.1355 18.5 27.1346C18.5 27.1337 18.5 27.1328 18.5 27.1318C18.5 27.1309 18.5 27.13 18.5 27.1291C18.5 27.1282 18.5 27.1273 18.5 27.1263C18.5 27.1254 18.5 27.1245 18.5 27.1236C18.5 27.1227 18.5 27.1218 18.5 27.1208C18.5 27.1199 18.5 27.119 18.5 27.1181C18.5 27.1172 18.5 27.1163 18.5 27.1154C18.5 27.1144 18.5 27.1135 18.5 27.1126C18.5 27.1117 18.5 27.1108 18.5 27.1099C18.5 27.1089 18.5 27.108 18.5 27.1071C18.5 27.1062 18.5 27.1053 18.5 27.1044C18.5 27.1035 18.5 27.1025 18.5 27.1016C18.5 27.1007 18.5 27.0998 18.5 27.0989C18.5 27.098 18.5 27.097 18.5 27.0961C18.5 27.0952 18.5 27.0943 18.5 27.0934C18.5 27.0925 18.5 27.0916 18.5 27.0906C18.5 27.0897 18.5 27.0888 18.5 27.0879C18.5 27.087 18.5 27.0861 18.5 27.0851C18.5 27.0842 18.5 27.0833 18.5 27.0824C18.5 27.0815 18.5 27.0806 18.5 27.0797C18.5 27.0787 18.5 27.0778 18.5 27.0769C18.5 27.076 18.5 27.0751 18.5 27.0742C18.5 27.0732 18.5 27.0723 18.5 27.0714C18.5 27.0705 18.5 27.0696 18.5 27.0687C18.5 27.0677 18.5 27.0668 18.5 27.0659C18.5 27.065 18.5 27.0641 18.5 27.0632C18.5 27.0623 18.5 27.0613 18.5 27.0604C18.5 27.0595 18.5 27.0586 18.5 27.0577C18.5 27.0568 18.5 27.0558 18.5 27.0549C18.5 27.054 18.5 27.0531 18.5 27.0522C18.5 27.0513 18.5 27.0504 18.5 27.0494C18.5 27.0485 18.5 27.0476 18.5 27.0467C18.5 27.0458 18.5 27.0449 18.5 27.0439C18.5 27.043 18.5 27.0421 18.5 27.0412C18.5 27.0403 18.5 27.0394 18.5 27.0385C18.5 27.0375 18.5 27.0366 18.5 27.0357C18.5 27.0348 18.5 27.0339 18.5 27.033C18.5 27.032 18.5 27.0311 18.5 27.0302C18.5 27.0293 18.5 27.0284 18.5 27.0275C18.5 27.0266 18.5 27.0256 18.5 27.0247C18.5 27.0238 18.5 27.0229 18.5 27.022C18.5 27.0211 18.5 27.0201 18.5 27.0192C18.5 27.0183 18.5 27.0174 18.5 27.0165C18.5 27.0156 18.5 27.0146 18.5 27.0137C18.5 27.0128 18.5 27.0119 18.5 27.011C18.5 27.0101 18.5 27.0092 18.5 27.0082C18.5 27.0073 18.5 27.0064 18.5 27.0055C18.5 27.0046 18.5 27.0037 18.5 27.0027C18.5 27.0018 18.5 27.0009 18.5 27H15.5ZM16.5181 26.9858C7.16386 30.1595 0.5 39.1651 0.5 49.5H3.5C3.5 40.4599 9.33614 32.5905 17.4819 29.8267L16.5181 26.9858ZM0.5 49.5C0.5 62.8284 11.1716 73.5 24.5 73.5V70.5C12.8284 70.5 3.5 61.1716 3.5 49.5H0.5ZM24.5 73.5H82V70.5H24.5V73.5ZM82 73.5C93.7583 73.5 103.5 63.9298 103.5 52H100.5C100.5 62.2577 92.1167 70.5 82 70.5V73.5ZM103.5 52C103.5 41.735 96.1025 33.0119 86.2052 30.9988L85.6073 33.9387C94.1475 35.6756 100.5 43.2025 100.5 52H103.5Z"/></symbol>
```

Comment:
This increases the size of this template from 7KB to 70KB and it needs to be rendered on every single page so this could have an impact on server performance. As they're only required on one part of the site, could these be moved into external SVG files?

Reply:
Good call @kaedroho 

I had moved between the icon being in the sprite and being stored separately a few times. In the end it seemed more consistent with the rest of the icons to store them in the sprite. 

But considering the performance impact, I am happy to move that our to a separate file. Maybe a second sprite for the icons used only on this page? 

I will also see if a SVG optimiser can help reduce the file size.

Reply:
This is implemented now. The new separated sprite is also reduced to 17KB.